spec/stat_tracker_spec.rb

```diff
@@ -39,107 +39,154 @@
   end
 
   describe "#highest_total_score" do
-    it "" do
+    it "finds the highest sum of the winning and losing teams scores" do
+      expect(@tracker.highest_total_score).to eq(11)
     end
   end
 
   describe "#lowest_total_score" do
-    it "" do
+    it "finds the lowest sum of the winning and losing teams scores" do
+      expect(@tracker.lowest_total_score).to eq(0)
     end
   end
 
   describe "#percentage_home_wins" do
```

Comment:
It looks like a few of these are still failing due to rounding errors (and multiplying by 100) - I'll check those out in a separate branch and get things updated where needed.